### PR TITLE
Add source rsaosago base alt

### PR DESCRIPTION
**Pull request by EddieOrbit**
### Added

- Source `rsaosago.base.alt`

### Changed

- Field with path `insurance.osago.items[].geo` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.date.update` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].date.end` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].owner.dob` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].owner.tin` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].date.start` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].owner.type` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].owner.name` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].insurer.name` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].date.created` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].contract.kbm` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].insurant.dob` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].insurant.tin` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].policy.series` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].policy.number` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].policy.status` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].insurant.type` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].insurant.name` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].policy.expired` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].policy.is_active` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].restrictions.type` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].contract.is_active` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].vehicle.model.name` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].vehicle.weight.max` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].restrictions.drivers` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].contract.has_trailer` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].contract.amount.value` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].vehicle.identifiers.vin` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].vehicle.engine.power.hp` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].contract.amount.currency` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].vehicle.identifiers.body` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].contract.using_type.status` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].vehicle.identifiers.reg_num` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].contract.using_type.description` also fillable by `rsaosago.base.alt`
- Field with path `insurance.osago.items[].contract.is_follow_to_registration` also fillable by `rsaosago.base.alt`

- Field with path `insurance.osago.items[].date.created` also fillable by `rsaosago.base.ext`

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I wrote unit tests for my code _(if tests is required for my changes)_
- [X] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,52 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## UNRELEASED
+
+### Added
+
+- Source `rsaosago.base.alt`
+
```
Comment (Zaboday) [inline CHANGELOG.md:12]: - Field with path `insurance.osago.items[].date.created`

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,52 @@ All notable changes to this project will be documented in this file.
...
+### Changed
+
+- Field with path `insurance.osago.items[].geo` also fillable by `rsaosago.base.alt`
```
Comment (Zaboday) [inline CHANGELOG.md:15]: - Field with path `insurance.osago.*` also fillable by `rsaosago.base.alt`

File: CHANGELOG.md (hunk 1 above)
Comment (EddieOrbit) [reply CHANGELOG.md:13]: Готово

File: CHANGELOG.md (hunk 2 above)
Comment (EddieOrbit) [reply CHANGELOG.md]: Готово

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
...
+## UNRELEASED
+
+### Added
+
+- Source `rsaosago.base.alt`
+- Field with path insurance.osago.items[].date.created
+
+### Changed
+
+- Field with path `insurance.osago.*` also fillable by rsaosago.base.alt
```
Comment (Zaboday) [inline CHANGELOG.md:16]: кавычки у источника:

- Field with path `insurance.osago.*` also fillable by `rsaosago.base.alt`

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## UNRELEASED
+
+### Added
+
+- Source `rsaosago.base.alt`
+- Field with path insurance.osago.items[].date.created
```
Comment (Zaboday) [inline CHANGELOG.md:12]: кавычки у названия поля:

- Field with path insurance.osago.items[].date.created

File: CHANGELOG.md (hunk 3 above)
Comment (EddieOrbit) [reply CHANGELOG.md]: Готово

File: CHANGELOG.md (hunk 4 above)
Comment (EddieOrbit) [reply CHANGELOG.md]: Готово

**Pull request merged**